samples/Localization.StarterWeb/Localization.StarterWeb.csproj

```diff
@@ -42,7 +34,6 @@
 
   <ItemGroup>
     <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="1.2.0-*" />
-    <DotNetCliToolReference Include="Microsoft.Extensions.SecretManager.Tools" Version="1.0.1-*" />
```

Comment:
What are the tools used for in this context? Project builds and runs fine and that's all we need for the sample.